src/components/sbb-checkbox/sbb-checkbox.tsx

```diff
@@ -115,6 +115,15 @@ export class SbbCheckbox implements ComponentInterface {
   })
   public stateChange: EventEmitter<CheckboxStateChange>;
 
+  /**
+   * Internal event that emits when the input element is loaded.
+   */
+  @Event({
+    bubbles: true,
+    eventName: 'sbb-checkbox-loaded',
+  })
+  public sbbCheckboxLoaded: EventEmitter<void>;
```

Comment:
Somehow weird with the sbb prefix, but also somehow ok :D.
As it is triggered in connected callback, should it better be named sbbCheckboxConnected? @kyubisation your meaning?